espefuse/efuse/emulate_efuse_controller_base.py

```diff
@@ -33,16 +27,16 @@ def __init__(self, efuse_file=None, debug=False):
         if self.efuse_file:
             try:
                 self.mem = BitStream(
-                    bytes=open(self.efuse_file, "rb").read(),
-                    length=self.REGS.EFUSE_MEM_SIZE * 8,
+                    open(self.efuse_file, "a+b"), length=self.REGS.EFUSE_MEM_SIZE * 8
                 )
-            except (ValueError, FileNotFoundError):
```

Comment:
Is the FileNotFoundError not possible or sufficiently unlikely?
Just curious why we open the file in binary append mode in different to binary read, although I'm not so familiar with bitstring/bitstream.